Form/EventListener/FixCheckboxDataListener.php

```diff
@@ -36,16 +36,18 @@ public static function getSubscribedEvents()
     }
 
     /**
+     * NEXT_MAJOR: remove this method.
```

Comment:
To be redundant ? I don't think so, especially given this is going to be removed afterwards… plus if we forget to grep for deprecated, we are covered… or maybe we should have used `@deprecated` everywhere instead of `NEXT_MAJOR` ?